backend/src/utils/stream.py

```
@@ -286,3 +286,61 @@ async def stream_generator(
+            # Block for up to 30 seconds waiting for messages
+            messages = await redis_client.xread(
+                {stream_key: last_id},
+                block=30000,  # 30 second timeout
+            )
+
+            if not messages:
+                # No messages yet, yield a keep-alive comment
+                yield ": keep-alive\n\n"
+                continue
+
+            for stream, entries in messages:
```

Ruff (0.14.10)

Loop control variable `stream` not used within loop body

Rename unused `stream` to `_stream`

(B007)